### ISSUE TITLE
Exterior Wall versus Roof insulation on cooling demand

### ISSUE THREAD

**Issue by RicardoGomesIST** [architecture-building-systems/CityEnergyAnalyst#2987]
Hi, 
I am performing a retrofit analysis on residential building stock. I am analyzing two retrofit solutions. 
Walls insulation and roof insulation.
As expected both resulted in lower heating demand. On the other hand, only roof insulation resulted on lower cooling demand and wall insulation resulted on worst (higher cooling demands) comparing to the original scenario.
I simply reduced the U value on the database for wall and roof solutions..
My HVAC availability is: from 10h to 24h during weekends and from 7h to 9h and 18h to 24h on weekdays. Setpoint 26ºC / Portuguese weather file. 

Is there explanation for this?
TY!

**Comment by lguilhermers** [comment]
@RicardoGomesIST from my experience, the relationship between U-values is not always straightforward, due to the thermal inertia of the envelope. 
Very insulated buildings can prevent the building to increase its indoor temperature when outside is hotter, but can also prevent the heat of coming out though the envelope when is outside is colder.
I would suggest testing different intermediate U-Values to have a better feeling about this, and also looking at the hourly energy balances to understand how it is affecting the loads.

**Comment by RicardoGomesIST** [comment]
Thank you for your answer. Can you please tell me where I can find the equations used for the calculation of the different variables, as for example:
I_sol_kWh
I_rad_kWh

Has CEA an document with all the physics behind it?
thank you

**Comment by lguilhermers** [comment]
Hi @RicardoGomesIST. I think the best way to check for all the underlying calculations for each variable would be to have the developer version installed and look into the code. Given the complexity of all the calculations underneath CEA, it is not really feasible to make into a written documentation. But the code should be well commented to make such understand easier.

Furthermore, I can anticipate that some of the calculations involved solar radiation are generated by using Daysim, a third party software that is used as dependency for CEA. Hopefully this helps you.

**Comment by shanshanhsieh** [comment]
Hi @RicardoGomesIST,

Here you can find the function that calculates `I_sol` and `I_rad`:
https://github.com/architecture-building-systems/CityEnergyAnalyst/blob/1e7274b6d9f8b15f00c6c8ce91c820b24b7fc39b/cea/demand/sensible_loads.py#L51-L107

all of the calculations were taken from standards, we will find time to compile these standards.
hope this helps for now.

**Comment by RicardoGomesIST** [comment]
Thank you!

**Comment by RicardoGomesIST** [comment]
I would like to briefly share my analysis. 
---- ASSUMPTIONS -----
considering:
residential buildings (occupied by working class):

- non insulated walls = 0.987
- Insulated walls =0.35
- non insulated roofs = 2.4697
- Insulated roofs= 0.493

-------------RESULTS ------------------
Low height buildings:
![image](https://user-images.githubusercontent.com/52715280/122901768-c23e5080-d345-11eb-92c9-740db3a51d2c.png)
![image](https://user-images.githubusercontent.com/52715280/122901825-d1250300-d345-11eb-8530-90415138f044.png)


                                                      | Low   height buildings (high roof-wall ratio)    | High   height (low roof-wall ratio)
                                                  | If  Insulated walls kWh | If  Insulated roofs kWh   | If  Insulated walls kWh | If  Insulated roofs kWh
I_sol_kWh   (Solar heat gain)        | 11464                         | 4750                              | 39068                             | 27754
I_rad_kWh   (Radiative heat loss)  | 6183                          | 4059                               | 22250                            | 21220
I_sol_and_I_rad_kWh   (balance)   | 5281                          | 690                                 | 16817                             | 6533


--------------CONCLUSIONS ------------------
- roof insulation reduces significantly the solar heat gain of one building, more than wall insulation. Of course this is more clear for buildings with higher roof-wall ratios. 
- insulation does not affect as much the radiative heat loss.

In conclusion the higher reduction on cooling loads for roof insulating is related with the reduction of the effect of solar gains on the roof by conduction.

**Comment by shanshanhsieh** [comment]
thanks for sharing the results, @RicardoGomesIST!
another point worth mentioning is that the original thermal transmittance of roof is much higher than wall, which could also be why the roof insulation leads to more reduction.
